iamrahulmahato/master-web-development
# ISSUE #1343 (solved) Add Image Carousel Project

This pull request adds the entire Image Carousel Project built with HTML, CSS, and JavaScript. The project showcases an interactive carousel that enables users to view images with smooth transitions and responsive design.

Objective:
Create a fully functional and responsive image carousel to enhance user interaction and improve visual presentation.

Features:

Manual Controls: Next and previous buttons for navigation.
Auto-Play Option: Images can rotate automatically.
Responsive Design: Works across different screen sizes and devices.
Lightweight: Built without external libraries.
Project Structure:

arduino
Copy code
/image-carousel-project
│
├── index.html       // Main HTML file
├── styles.css       // CSS for styling the carousel
└── script.js        // JavaScript for functionality
How to Test:

Clone the repository.
Navigate to the image-carousel-project/ directory.
Open index.html in any browser to view the project.
Test manual controls and resize the window to verify responsiveness.
Checklist:

 All project files added.
 Code is well-documented and structured.
 Responsive across devices.
 No console warnings or errors.

ScreenShot:
![Screenshot 2024-10-17 163700](https://github.com/user-attachments/assets/cbbd48e3-e00b-4b9d-ad68-5d97053d27f0)
